docs/serving/autoscaling/autoscaling-concepts.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Autoscaling concepts"
+linkTitle: "Autoscaling concepts"
+weight: 01
+type: "docs"
+---
+
+This section covers conceptual information about which Autoscaler types are supported, as well as fundamental information about how autoscaling is configured.
+
+## Supported Autoscaler types
+
+Knative Serving supports the implementation of Knative Pod Autoscaler (KPA) and Kubernetes' Horizontal Pod Autoscaler (HPA). The features and limitations of each of these Autoscalers are listed below.
+
+**IMPORTANT:** If you want to use Kubernetes Horizontal Pod Autoscaler (HPA), you must install it after you install [Knative Serving](../../install/any-kubernetes-cluster.md#optional-serving-extensions).
+
+### Knative Pod Autoscaler (KPA)
+
+* Part of the Knative Serving core and enabled by default once Knative Serving is installed.
+* Supports scale to zero functionality.
+* Does not support CPU-based autoscaling.
+
+### Horizontal Pod Autoscaler (HPA)
+
+* Not part of the Knative Serving core, and must be enabled after [Knative Serving installation](../../install/any-kubernetes-cluster.md#optional-serving-extensions).
+* Does not support scale to zero functionality.
+* Supports CPU-based autoscaling.
+
+### Configuring the Autoscaler implementation
+
+The type of Autoscaler implementation (KPA or HPA) can be configured by using the `class` annotation.
+
+* **Global settings key:** `pod-autoscaler-class`
+* **Per-revision annotation key:** `autoscaling.knative.dev/class`
+* **Possible values:** `"kpa.autoscaling.knative.dev"` or `"hpa.autoscaling.knative.dev"`
+* **Default:** `"kpa.autoscaling.knative.dev"`
+
+**Example:**
+{{< tabs name="class" default="Per Revision" >}}
+{{% tab name="Per Revision" %}}
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: helloworld-go
+  namespace: default
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/class: "kpa.autoscaling.knative.dev"
+    spec:
+      containers:
+        - image: gcr.io/knative-samples/helloworld-go
+```
+{{< /tab >}}
+{{% tab name="Global (ConfigMap)" %}}
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+ name: config-autoscaler
+ namespace: knative-serving
+data:
+ pod-autoscaler-class: "kpa.autoscaling.knative.dev"
+```
+{{< /tab >}}
+{{% tab name="Global (Operator)" %}}
+```yaml
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+spec:
+  config:
+    autoscaler:
+      pod-autoscaler-class: "kpa.autoscaling.knative.dev"
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+## Global versus per-revision settings
```

Comment:
This should probably go above the class, since it affects the choices made there as well.

Reply:
I think this fits here as well -- it provides more detail, but if users are happy with the first choice they found, it should work.

Reply:
I think this is why we need to be able to reuse / include content - so we can have a blurb about something in more than one place in the docs but maintain it from one file.